### ISSUE TITLE
Implement Upload in Plugin

### ISSUE THREAD

**Issue by jneilliii** [jneilliii/OctoPrint-CustomBackground#44]
In order to get around potential future complaints relative to Slicer Thumbnails and using the uploads folder for extracted images, incorporate an internal uploads option in settings instead of using the file manager in OctoPrint to upload. See https://github.com/jneilliii/OctoPrint-PrusaSlicerThumbnails/issues/89#issuecomment-1336454047.

**Comment by bolsoncerrado** [comment]
Reviving request.
Perhaps a temp workaround to HIDE any non .gcode files on the file list setting in the meantime?

**Comment by jneilliii** [comment]
yeah, I'll get to it eventually. Just have a bunch of plugins that I have to make OctoPrint 1.9.0 compatible first.

**Comment by bolsoncerrado** [comment]
Awesome :P
